irohad/network/impl/block_loader_impl.cpp

```diff
@@ -17,9 +17,10 @@
 
 #include "network/impl/block_loader_impl.hpp"
 
+#include <grpc++/create_channel.h>
 #include <algorithm>
```

Comment:
Look like broken clang format

Reply:
It did not complain, but I still reordered include lines